charters/ExpressionMatrixService/charter.md

```diff
@@ -0,0 +1,107 @@
+
+# Expression Matrix Service
+
+## Description
+
+The expression matrix service enables researchers to easily access and analyze HCA expression matrix data.
+
```

Comment:
Can you add some description about what the service specifically does; ie. something about how it takes requests from users for specific data and delivers a matrix of expression values.. in some format or formats. The sentence here just says why it's useful.

Reply:
@jodihir - This is covered in the **Objectives** that follow, but I agree the **Description** is a bit of koan. I could ~steal~ borrow from a document that @mckinsel has been writing:

_The Human Cell Atlas (HCA) Data Coordination Platform (DCP) ingests, processes, and stores single-cell data, making that data easily available to a wide audience of researchers. The single-cell data type of most interest to researchers is the expression matrix, the cell-by-gene expression values that are the starting point for many downstream analyses. To support this data type, the Expression Matrix Service defines a public API to return expression matrices in response to queries._

Thoughts?

Reply:
sounds good